### DISCUSSION TITLE
test discussion / 6 Jun 2023

### DISCUSSION THREAD

**Discussion by vetter** [ORNL/iris#12] [General]
Folks. are you able to receive notifications from this discussion thread? should we use it for iris design discussions?

**Comment by vetter** [comment]
@ORNL/iris-dev : addressing the iris-dev group.

**Comment by Geekdude** [comment]
Hi Jeff, I received an email for both the initial discussion creation and the comment.

I enabled the discussion feature of this GitHub repo so that we would have a public place to discuss IRIS that is separate from reporting issues. I have also updated the README to include using discussions as a method to get in touch with the IRIS team.
